src/main/java/com/ericsson/ec/helpers/JenkinsXmlData.java

```diff
@@ -187,20 +180,17 @@ private void validatePropertiesObject(String parametertypeKey) {
             properties.put(HUDSON_PARAMETERS_DEFINITION_KEY, parametersDefinitionProperty);
         }
 
-        boolean hasParameterDefinitions = properties.getJSONObject(HUDSON_PARAMETERS_DEFINITION_KEY)
-                .has(PARAMETER_DEFINITION_KEY);
+        boolean hasParameterDefinitions = properties.getJSONObject(HUDSON_PARAMETERS_DEFINITION_KEY).has(PARAMETER_DEFINITION_KEY);
```

Comment:
This line maybe should not be so long?

Reply:
Not according to the eiffel formatter in confluence. We are likely using different versions.